src/race.rs

```diff
@@ -5,6 +5,18 @@
 //! them stores the result.
 //!
 //! This module does not require `std` feature.
+//!
+//! # Atomic orderings
+//!
+//! All types in this module use `Acquire` and `Release`
+//! [atomic orderings](Ordering) for all their operations. While this is not
+//! strictly necessary, it is useful for users as it allows them to be certain
```

Comment:
This actually **is** strictly necessary for `OnceBox` I believe. Othewise, you could leak partialy constructed objects

Reply:
Yeah, I think you might be right. I'll update it once I have access to a computer.